docs/build_arm.md

```diff
@@ -46,24 +96,18 @@ sudo apt-get install crossbuild-essential-armhf
 ```
 On non-Debian based systems, the package could be called `arm-linux-gnueabihf`.
 
-To build for ARM32 architecture, run the following command from
-the LCE root directory:
+To build for 32-bit ARM architectures, run the following command from the LCE
+root directory:
 ```bash
 larq_compute_engine/tflite/build_make/build_lce.sh --rpi
 ```
-When building for ARM64 architecture, replace `--rpi` with `--aarch64`.
+When building for a 64-bit ARM architecture, replace `--rpi` with `--aarch64`.
 
-### Compile natively with Make ###
-To natively build the LCE library and C++ example programs,
-first you need to install the compiler toolchain on your target device.
-For example, on a Raspberry Pi board, run the following command:
-```
-sudo apt-get install build-essential
-```
-
-You should then be able to natively compile LCE:
-```bash
-larq_compute_engine/tflite/build_make/build_lce.sh --native
-```
-NOTE: when building natively on a 32-bit Raspberry Pi, replace `--native` with
-`--rpi`.
+Just as with a native build, the resulting compiled files will be stored in
```

Comment:
this is the duplicate of the paragraph above. why not just say the same as above?

Reply:
Yes, thats better.